Assets/MixedRealityToolkit/Utilities/ClippingSphere.cs

```diff
@@ -12,14 +12,17 @@ namespace Microsoft.MixedReality.Toolkit.Utilities
     [ExecuteInEditMode]
     public class ClippingSphere : ClippingPrimitive
     {
-        [Tooltip("The radius of the clipping sphere.")]
-        [SerializeField]
-        protected float radius = 0.5f;
-
         /// <summary>
-        /// The radius of the clipping sphere.
+        /// The radius of the clipping sphere, which is determined by the largest axis of the transform's scale.
         /// </summary>
-        public float Radius => radius;
+        public float Radius
```

Comment:
Can you also add a note to:

https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/UpdatingToGA.html

This is technically a breaking change because we're changing the shape of our API to one that is explicit (i.e. passing a radius via config or runtime setting) to one that is implicit based on the transform of the tracked object. It seems like this one seems more useful especially for clipping sphere, but it would be good to call out the change there (i.e. before  you would have to track the transform and update the radius yourself, now it automatically tracks it, if you want to change it, just update the transform of the sphere as you normally would.)

Reply:
Thank you for reviewing, I’ll update this document tomorrow!